src/math/vec2.js

```diff
@@ -429,6 +429,18 @@ class Vec2 {
         return '[' + this.x + ', ' + this.y + ']';
     }
 
+    /**
+     * @function

```

Comment:
Same, let's keep this private for now and just use this function for internal purposes.